### PR TITLE
Replace bg images with css, and add fallback bg colors

### PR DESCRIPTION
**Pull request by boshevski**
Improvements on website loading times by replacing background gradients with css code.
Also adding fallback background colors where missing.

### CODE REVIEW

**Comment by boshevski** [comment]
Forgot to add these updates in the description:
1. Updated partners link in the footer to point to partner fork page.
2. Added how it works page to the footer.

File: assets/scss/_single-connector.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .single-connector {
   &__nav {
+    background: #0059d0;
```
Comment (mbouzi) [inline assets/scss/_single-connector.scss]: maybe we can turn this into a variable?

**Review by mbouzi — CHANGES_REQUESTED** [review]

File: assets/scss/_single-connector.scss (hunk 1 above)
Comment (boshevski) [reply assets/scss/_single-connector.scss]: Yeah, great call Malcolm! We actually have this already as a variable. I'll fix it :)

**Review by mbouzi — APPROVED** [review]
👍

**Pull request merged**